webhooks/assign-class-label/certificate.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: assign-class-label-tls
+spec:
+  secretName: assign-class-label-tls
+  isCA: true
```

Comment:
Is this certificate being used as a CA? Or is it used as a regular server certificate?

Reply:
AFAIK it's a server certificate - we can probably remove the `isCA` or set it to false.